src/app/components/Item/ItemTableRow.jsx

```diff
@@ -15,12 +15,11 @@ const ItemTableRow = ({ item, bibId, getRecord }) => {
 
   if (item.requestHold) {
     itemLink = item.available ?
-      <Link
-        className="request-button"
+      <button
```

Comment:
The `to` attribute is not valid for the `<button>` element. Why the change to a button instead of a link?